spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -130,6 +130,25 @@
     end
 
     context 'user is not signed in' do
+      context 'without valid acr_values' do
+        before { params.delete(:acr_values) }
+
+        it 'handles the error and does not blow up' do
+          action
+
+          expect(response).to redirect_to(/^#{params[:redirect_uri]}/)
```

Comment:
Some requests are "good enough" that we can redirect and provide an error to the client, so that they can handle the error (good enough in that the `client_id` and `redirect_uri` match each other)

We display errors when the requests are not good enough to redirect